### DISCUSSION TITLE
Select Sum and Average example

### DISCUSSION THREAD

**Discussion by Faridalim** [piccolo-orm/piccolo#141] [Q&A] [answered]
Hi, could you give me example of how to select sum and average of a column?

**Comment by sinisaos** [comment, marked as answer]
@Faridalim Currently you need to use Piccolo’s [raw](https://piccolo-orm.readthedocs.io/en/latest/piccolo/query_types/raw.html) clause for aggregation. Here is an [example](https://github.com/sinisaos/starlette-piccolo-rental/blob/master/ads/endpoints.py). I hope this helps you.

**Reply by sinisaos** [reply]
@Faridalim Now you can use the new [aggregate functions](https://piccolo-orm.readthedocs.io/en/latest/piccolo/query_types/select.html#aggregate-functions) directly in Piccolo. For complex aggregation included multiple columns at once, you still can use Piccolo [raw](https://piccolo-orm.readthedocs.io/en/latest/piccolo/query_types/raw.html) clause.

**Reply by Faridalim** [reply]
Great. Thank you @sinisaos..

**Reply by sinisaos** [reply]
@Faridalim You're welcome. I'm glad I can help.
